e2e/staticpvc.go

```diff
@@ -53,6 +63,56 @@ func getStaticPV(
 	return pv
 }
 
+func getMigrationStaticPV(
```

Comment:
cant we reuse the `getStaticPV` and add/update what is required for migration instead of recreating the PV object?

Reply:
Done.